x/concentrated-liquidity/tick_test.go

```diff
@@ -298,35 +298,65 @@ func (s *KeeperTestSuite) TestInitOrUpdateTick() {
 func (s *KeeperTestSuite) TestGetTickInfo() {
 	var (
 		preInitializedTickIndex = DefaultCurrTick.Int64() + 2
+		expectedUptimes         = getExpectedUptimes()
+		emptyUptimeTrackers     = wrapUptimeTrackers(expectedUptimes.emptyExpectedAccumValues)
+		varyingTokensAndDenoms  = wrapUptimeTrackers(expectedUptimes.varyingTokensMultiDenom)
 	)
 
 	tests := []struct {
-		name             string
-		poolToGet        uint64
-		tickToGet        int64
-		expectedTickInfo model.TickInfo
-		expectedErr      error
+		name                     string
+		poolToGet                uint64
+		tickToGet                int64
+		preInitUptimeAccumValues []sdk.DecCoins
+		expectedTickInfo         model.TickInfo
+		expectedErr              error
 	}{
 		{
 			name:      "Get tick info on existing pool and existing tick",
 			poolToGet: validPoolId,
 			tickToGet: preInitializedTickIndex,
-			// Note that FeeGrowthOutside is not updated.
-			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg()},
+			// Note that FeeGrowthOutside and UptimeGrowthOutside(s) are not updated.
+			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg(), UptimeTrackers: emptyUptimeTrackers},
+		},
+		{
+			name:                     "Get tick info on existing pool and existing tick with init but zero global uptime accums",
+			poolToGet:                validPoolId,
+			tickToGet:                preInitializedTickIndex,
+			preInitUptimeAccumValues: expectedUptimes.varyingTokensMultiDenom,
+			// Note that FeeGrowthOutside is not updated, but UptimeGrowthOutsides are.
+			// We expect uptime trackers to be initialized to zero since tick > active tick
+			expectedTickInfo: model.TickInfo{LiquidityGross: DefaultLiquidityAmt, LiquidityNet: DefaultLiquidityAmt.Neg(), UptimeTrackers: emptyUptimeTrackers},
+		},
+		{
+			name:                     "Get tick info on existing pool and existing tick with nonzero global uptime accums",
+			poolToGet:                validPoolId,
+			tickToGet:                preInitializedTickIndex - 3,
```

Comment:
nit: maybe make a var with a name explaining that this tick is <= active tick, since -3 feels random / not explained